lms/envs/common.py

```diff
@@ -125,6 +125,12 @@
     # Can be turned off if course lists need to be hidden. Effects views and templates.
     'COURSES_ARE_BROWSABLE': True,
 
+    # Extra message to display in the Learner Dashboard when they are not enrolled in any courses yet.
+    'EMPTY_DASHBOARD_MESSAGE': None,
```

Comment:
Nit: I assume that since this is always going to be a (HTML) string (or is that incorrect?), we can default this to `''` to make it more straightforward for the user?